kpet/data.py

```diff
@@ -291,6 +291,7 @@ def __init__(self, data):
                     waived=Boolean(),
                     role=String(),
                     environment=Dict(String()),
+                    maintainers=List(String(), min_len=1),
```

Comment:
I think allowing specifying an empty list of maintainers makes it less cumbersome to edit or generate test cases. The field is optional anyway. Could you make it just `List(String())`?